tests/_utils/run_services

```diff
@@ -111,6 +122,25 @@ start_services() {
     done
 }
 
+start_tiflash() {
+    echo "Starting TiFlash..."
+    LD_LIBRARY_PATH=bin/ bin/tiflash server --config-file=$TIFLASH_CONFIG &
+    echo "TiFlash started..."
+
+    i=0
+    # FIXME: Add retry to makesure TiFlash start here, currently, seems pd-ctl NEVER returns at this context.
+    while false # ! bin/pd-ctl store | grep "tiflash"
```

Comment:
We can curl tiflash status page (it's the page from an embeded tikv in tiflash).

https://github.com/pingcap/ticdc/blob/33f568e6556e047ab6dacb42d4ac0ce5ef7b53a3/tests/_utils/start_tidb_cluster#L250

Reply:
That's a smart solution.

BTW, I copied the way CDC making TiFlash configuration files :smile:.